vslib/src/SwitchStateBase.cpp

```diff
@@ -2593,28 +2591,16 @@ sai_status_t SwitchStateBase::set_system_port_list()
     return set(SAI_OBJECT_TYPE_SWITCH, m_switch_id, &attr);
 }
 
-sai_status_t SwitchStateBase::set_voq_switch_attributes(
-        _In_ int32_t voq_switch_id,
-        _In_ uint32_t voq_max_cores)
+sai_status_t SwitchStateBase::refresh_system_port_list(
+        _In_ const sai_attr_metadata_t *meta)
 {
     SWSS_LOG_ENTER();
 
-    SWSS_LOG_INFO("set voq switch attributes");
-
-    sai_attribute_t attr;
-
-    attr.id = SAI_SWITCH_ATTR_TYPE;
-    attr.value.s32 = SAI_SWITCH_TYPE_VOQ;
-
-    CHECK_STATUS(set(SAI_OBJECT_TYPE_SWITCH, m_switch_id, &attr));
-
-    attr.id = SAI_SWITCH_ATTR_SWITCH_ID;
-    attr.value.u32 = voq_switch_id;
+    // Currently, system ports info are not changing and also since currently the local
+    // port oid for local system ports is retrieved only during portsorch initialization,
+    // no referesh is done
 
-    CHECK_STATUS(set(SAI_OBJECT_TYPE_SWITCH, m_switch_id, &attr));
+    // In the future, when dynamic system port config (add/delete) is implemented, re-visit TODO
```

Comment:
this is ok, but please put TODO string in front: "//  TODO" its is done in all files all over the project, keep consistent

Reply:
o.k. I'll fix

Reply:
Fixed